src/modals/GeolocationModal.jsx

```diff
@@ -88,4 +103,20 @@ export async function openPendingDirectionModal() {
   });
 }
 
+export async function openPendingGeolocateModal() {
+  return new Promise(resolve => {
+    open(
+      'PENDING',
+      () => {
+        close();
+        resolve(false);
```

Comment:
I think I saw in the designs that closing this modal here was equivalent to the `Continue` button. The principle is that the action was initiated by the user so they won't cancel it here.
You should check the spec for that.

Reply:
yep, however, the only usage of `openPendingGeolocateModal` (for now) ignore the returned value (accepted if true, closed if false). So this is the current behavior :)